tin/apps/venvs/tasks.py

```diff
@@ -15,6 +15,9 @@
 @shared_task
 def create_venv(venv_id):
     venv = Venv.objects.get(id=venv_id)
+    python = settings.SUBMISSION_PYTHON
```

Comment:
Can you either hardcode this (just like in `submissions/tasks.py`) or make a new env var for it (like `settings.VENV_BASE_PYTHON`)?